<launch>
	<node name="node_1" pkg="beginner_tutorials" type="uwbtest2" output="screen">
		<param name="node_id5" value="1" type="int"/>
		<param name="node_id4" value="2" type="int"/>
		<param name="node_id2" value="4" type="int"/>
		<param name="node_id6" value="5" type="int"/>
		<param name="node_id3" value="3" type="int"/>
		<param name="serial_port" value="/dev/ttyACM0" type="string"/>
	</node>
</launch>
